wallet_transactions/wallets/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.dispatch import receiver
+from django.db.models.signals import pre_save
+from django.conf import settings
+
+import random
+import string
+
+from .list_for_model import TYPEOFWALLET, CURRENCYS
+
+User = settings.AUTH_USER_MODEL
+COUNTOFWALLETYOUCANCREAT = 5
+
+
+class Wallet(models.Model):
+    """Describe table Wallet"""
+
+    name = models.CharField(blank=True, null=True, editable=False)
```

Comment:
it's a unique value. please add this attribute

Reply:
add unique attribute